pkg/utils/utils.go

```diff
@@ -157,7 +157,7 @@ func GetIVDPETMFromParamsMap(params map[string]interface{}, logger logrus.FieldL
 
 	ivdPETM, err := ivd.NewIVDProtectedEntityTypeManagerFromURL(&vcUrl, s3URLBase, insecure, logger)
 	if err != nil {
-		logger.Errorf("Error at creating new IVD PETM")
+		logger.Errorf("Error at creating new IVD PETM with error message %v", err)
```

Comment:
Let's give the vcURL and s3URLBase here so we know what it was trying to talk to